### DISCUSSION TITLE
Could not install pipenv

### DISCUSSION THREAD

**Discussion by iabhaychandel** [Atri-Labs/atrilabs-engine#656] [Help: Installation & Start]
I followed video and after having unsuccessful installation I followed docs but still I'm not able to install "pipenv"

**Comment by Somilvaishya** [comment]
same things here

**Comment by cruxcode** [comment]
Please elaborate more on what is the issue you are facing while installing?

**Reply by Somilvaishya** [reply]
when I installing Atri Cli I showed this
![WhatsApp Image 2023-02-08 at 11 04 36](https://user-images.githubusercontent.com/123815919/217443506-b2d1c23b-d9e7-4e49-9d73-209ab8d4b4fb.jpg)

**Reply by Jatin7726** [reply]
i am also facing the same error, everything installed but when comes to pipen packages it shows the same error.

**Reply by Somilvaishya** [reply]
I resolved my problem with visual code.

**Reply by Somilvaishya** [reply]
![Screenshot_20230209_113959](https://user-images.githubusercontent.com/123815919/217736618-e490e4a4-cd03-4c89-bf37-b927820eb93d.png)
now the Atri editor was not open

**Reply by darshitac11** [reply]
Check the solution provided here: https://github.com/Atri-Labs/atrilabs-engine/discussions/588
